controllers/userControllers.js

```diff
@@ -68,12 +68,12 @@ exports.updateUser = async (req, res) => {
     },
     { where: { id } }
   );
-  res.send("Dados de usuário atualizados");
+  res.send("User data updated");
 };
 
 exports.deleteUser = (req, res) => {
   const deletedUser = database.User.destroy({
     where: { id: req.params.id },
   });
-  res.send("Usuário deletado com sucesso");
+  res.send("User deleted with success");
```

Comment:
This line has nothing to do with create user. Its ok to leave it, but try not to mix things on a PR.

Reply:
Alright, I'll make a mental note to avoid doing this again in the future